indiweb/main.py

```diff
@@ -47,6 +47,8 @@
 parser.add_argument('--server', '-s', default='standalone',
                     help='HTTP server [standalone|apache] (default: standalone')
 
+parser.add_argument('--reboot', '-r', help='reboot system')
```

Comment:
I don't think we should have an arg to reboot when starting indiweb server. The API call is enough.